scripts/scrape-metrics.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
```

Comment:
we don't need this script anymore, right?

Reply:
Yes

Reply:
Will remove it